Crane/app/src/main/res/values/strings.xml

```diff
@@ -39,6 +39,8 @@
     <string name="select_destination_to_caption">To</string>
     <string name="select_dates">Select Dates</string>
     <string name="error_max_people">Error: We don\'t support more than %d people</string>
+    <string name="tint_transition_spec">tintTransitionSpec</string>
+    <string name="tint_transition">tintTransition</string>
```

Comment:
This doesn't have to be part of the strings.xml as this information is just for the tooling, not for the app.
You can just keep the strings directly in the code.